Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/rapidantigen/RapidAntigenProcessor.kt

```diff
@@ -28,7 +28,7 @@ import javax.inject.Inject
 @Reusable
 class RapidAntigenProcessor @Inject constructor(
     private val timeStamper: TimeStamper,
-    private val submissionService: CoronaTestService,
+    private val submissionService: CoronaTestService
```

Comment:
Please re-add the commas. Since we upgraded Kotlin we can use the trailing commas, and they prevent merge conflicts if multiple people add new lines to the constructor.